src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
+
+Everything is well supported on this laptop and install will work without problems. Post-installation, after having booted to your newly installed system, there is however a few points that need a little bit of work.
+
+## keymap are wrong
+This is a problem inherited by a few distros, including archlinux.
+tty keymaps for the macbooks (mac-fr, mac-us, mac-de) are completely broken and *you should not attempt to use them*, or it will result in an unusable keyboard and you will be forced to do a hard reset on your system. as a workaround generic keymaps
+work well (fr, us, de ..) you can use those for starter. later on you can find working ones and install/use them. here i've found a working one for the french macbook keyboards: [forums.archlinux.fr](https://forums.archlinux.fr/viewtopic.php?t=8679)
```

Comment:
The documentation doesn't have an "I," so "I've found" isn't the tone that documentation should have. Random forum posts aren't a great source of information here either. If asking people to install something, it really needs to be packaged.

Reply:
please do so. i'm just struggling to make it work nicely, and i just thought about sharing my progress in the docs, but packaging working keymaps would be indeed better.